command/format.go

```diff
@@ -125,6 +126,27 @@ func (j JsonFormatter) Output(ui cli.Ui, secret *api.Secret, data interface{}) e
 	return nil
 }
 
+// An output formatter for raw output of the original request object
+type RawFormatter struct{}
+
+func (r RawFormatter) Format(data interface{}) ([]byte, error) {
+	byte_data, ok := data.([]byte)
+	if !ok {
+		return nil, fmt.Errorf("unable to type assert to []byte: %T; please share the command run", data)
```

Comment:
nit: I'm not sure how serviceable this error message is, from a user's perspective. I'm the farthest thing from a wordsmith, but maybe it's enough just to say something about malformed data? As you said in your previous comment re: the panic that used to be here, this probably should never happen, so this quibble might be meaningless, but I wanted to mention it anyway.

Reply:
Yeah, I wanted to try and encourage them to share the command run with us though, so we could investigate it and try to reproduce it. Its not really an error on their side that's preventable AFAICT.

Reply:
But yeah, maybe I'll just do a direct type assertion and let Go panic instead.

Reply:
I think that's fair.